crates/bevy_ecs/src/system/function_system.rs

```diff
@@ -396,7 +396,7 @@ macro_rules! impl_build_system {
                 Input: SystemInput,
                 Out: 'static,
                 Marker,
-                F: FnMut(In<Input>, $(SystemParamItem<$param>),*) -> Out
+                F: FnMut(Input, $(SystemParamItem<$param>),*) -> Out
```

Comment:
After #15184, the input parameter no longer have to be wrapped in a `In`

Reply:
Interesting. Can we remove `In` completely in a follow-up then?